core/interception.js

```diff
@@ -454,6 +454,8 @@ NodeHandler.onMessageNodeReceived = async function(currentNode, messageNodes, is
         if (!isAllowed) break;
     }
 
+    await interceptViewOnceMessages(encNodes, messageId);
```

Comment:
This should be inside `onE2EMessageNodeReceived`, like in `checkForMessageDeletionNode`